chain/network/src/peer_manager/network_state/mod.rs

```diff
@@ -303,4 +352,64 @@ impl NetworkState {
             .with_span_context(),
         );
     }
+
+    /// Check for edges indicating that
+    /// a) there is a peer we should be connected to, but we aren't
+    /// b) there is an edge indicating that we should be disconnected from a peer, but we are connected.
+    /// Try to resolve the inconsistency.
+    pub fn fix_local_edges(self: &Arc<Self>, clock: &time::Clock) {
```

Comment:
Let's add  context to the comment that we just run this every minute.

Reply:
done